### PR TITLE
Allow access to mounted drives on KDE

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

File: com.usebottles.bottles.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ finish-args:
 - --filesystem=~/.local/share/bottles:create
 - --device=all
 - --filesystem=host
```
Comment (mirkobrombin) [inline com.usebottles.bottles.yml:40]: Shouldn't this be enough?

Comment (soredake) [reply com.usebottles.bottles.yml:40]:
```
[📦 com.usebottles.bottles ~]$ LC_ALL=C ls /media
ls: cannot access '/media': No such file or directory
```
Unfortunately no.

Comment (soredake) [reply com.usebottles.bottles.yml:40]: Seems this is a bug in flatpak, as it should share /media with `host` permission. https://github.com/flatpak/flatpak/blob/1.10.1/doc/flatpak-metadata.xml#L223-L234

Comment (soredake) [reply com.usebottles.bottles.yml:40]: Reported https://github.com/flatpak/flatpak/issues/4163

Comment (soredake) [reply com.usebottles.bottles.yml:40]: I've reinstall bottles and now /media is visible, strange.

**Pull request closed without merging**